meetings/2021-12-01.md

```diff
@@ -0,0 +1,124 @@
+* Elf: I think the important bit is the user having one UI or trusted App to set permissions. One place where they set them and they understand how it works and the consequences. The authz agent is the same sort of thing.
+* Henry: Perhaps one should start with questions. Your main claim is one app to do all authz (but it can do it only on one server).
+* Elf: Alice's authz agent would be aware of all the access granted to and by her.
+* ...: Both a launcher app and authz agent would have similar roles, but instead of living on a device, the agent would be a service in the cloud.
+* Henry: Let's keep in mind the follow your nose approach. For example if access was granted to the friend of a friend.
+* Elf: If access was not explicitly granted, there's still a question of discovery. They are slightly different scenarios.
+* Henry: What you're doing is similar to an inbox.
+* Elf: Yeah, in interop there are access receipts and access inbox. From thereon we use the notification system such as webhooks. Because there are problem with public inboxes.
+* Henry: I think from a WAC/ACP perspective, what Matthieu and I need is a brief presentation of exactly what you need from an interop perspective. And then if we know precisely what it is, we can have a proposal.
+* Emelia: From a cursory interop spec review, it seems that there's a lot that is re-defined (or already defined in authz such as ACP).
+* Elf: We need to collapse things on Agent rather than collapsing on resource.
+* Emelia: Might be worth mentioning that currently, Inrupt's solid client defines ACP as a separate resource. I'm not sure it might collapse as you say.
+* Elf: If you have a memo resource, the ACR of that would say what agents can do on them.
+* Matthieu: Yes, collapsing per user would be super expensive as per current implementation of ACP.
+* Elf: Yeah and data grants do it the other way.
+* Henry: But why would you need a different ontology?
+* ...: There is a way to express the same kind of thing with the WAC or the ACP ontology.
+* ...: It's a problem because everytime you're creating a new ontology to say the same thing, it's clutter. The first thing would be good to know whether you can just use something existing to say the same thing.
+* ...: If interop is about interoperability, it would be good that it reuses existing ontologies rather than creating a new one.
+* Elf: We reuse access modes.
+* Henry: Why the new predicate?
+* Elf: because the access modes are object of a statement where the subject is an access grant.
+* Henry: One might do a mapping from WAC to this ontology.
+* ...: I've used the accessTo predicate for many things and I think it's there in the ontology.
+* ...: I think if we could simplify it would be really useful.
```

Comment:
```suggestion
* Elf: because the access modes are the objects of a statement where the subject is an access grant.
* Henry: One might do a mapping from WAC to this ontology.
  * I've used the `accessTo` predicate for many things, and I think it's there in the ontology.
  * I think it would be really useful if we could simplify.
```